WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -987,6 +987,10 @@ function(params) {
 
 	var html = params.html || "";
 	
+	if(html.search(/(<form)(?![^>]+action)(.*?>)/g)){
+        html=html.replace(/(<form)(?![^>]+action)(.*?>)/g, `$1 action="SAMEHOSTFORMPOST-BLOCKED" target="_blank"$2`)
```

Comment:
we should remove target attribute if already present
currently it does something like this `<form action="SAMEHOSTFORMPOST-BLOCKED" target="_blank" target="inline"></form>` if target is already present

Reply:
Resolved